apps/web/app/Projects/escrows/page.tsx

```
@@ -0,0 +1,236 @@
+				})
+				.eq('id', id)
+
+			if (error) throw error
+
+			await fetchRecords()
+			alert(`Status updated to ${newStatus}`)
+		} catch (err) {
+			console.error('Error:', err)
+			alert(
+				'Error inserting test data: ' +
+					(err instanceof Error ? err.message : 'Unknown error'),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)